docs/user-guide/bandwidth-resources.md

```diff
@@ -0,0 +1,170 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+<strong>
+The latest 1.0.x release of this document can be found
+[here](http://releases.k8s.io/release-1.0/docs/user-guide/bandwidth-resources.md).
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+# Bandwidth Resources
+
+When specifying a [pod](pods.md), you can optionally specify how much bandwidth (bits/sec) each
+container can consume.  There are separate constraints for ingress and egress bandwidth. When pods
+have resource limits, the scheduler is able to make better
+decisions about which nodes to place pods on, and contention for resources can be handled in a
+consistent manner.
+
+*ingress* and *egress* bandwidth are each a *resource type*.  All resource types have a base unit.
+For both ingress and egress this unit is bits per second.
+
+Ingress and egress are collectively referred to as *bandwidth resources*.  Bandwidth
+resources are measureable quantities which can be requested, allocated, and consumed.  They are
+distinct from [API resources](working-with-resources.md).  API resources, such as pods and
+[services](services.md) are objects that can be written to and retrieved from the Kubernetes API
+server.
+
+## Pod Resource Limits
+
+Each Pod can optionally specify `spec.containers[].resources.limits.ingress` and/or
```

Comment:
What if the user specified request? How will request be handled at the node level?